Essentials/src/messages_en.properties

```diff
@@ -421,6 +421,7 @@ powerToolRemove=§6Command §c{0}§6 removed from §c{1}§6.
 powerToolRemoveAll=§6All commands removed from §c{0}§6.
 powerToolsDisabled=§6All of your power tools have been disabled.
 powerToolsEnabled=§6All of your power tools have been enabled.
+priceChanged=§6Price changed from {0} to {1}.
```

Comment:
Please do not add this here, Crowdin will automatically do this
```suggestion
```

Reply:
didn't know that